maven-publish.gradle

```diff
@@ -0,0 +1,95 @@
+apply plugin: "maven-publish"
+apply plugin: "signing"
+
+/*
+variables:
+ext.mavenPublishProperties = [
+        group      : "co.infinum",
+        version    : versions.prince,
+        artifact   : "prince-of-versions",
+        repository : [
+                url     : sonatype.url,
+                username: sonatype.username,
+                password: sonatype.password
+        ],
+        name       : "Prince of Versions",
+        description: "Library that checks for updates using configuration from remote or local resources",
+        url        : "https://github.com/infinum/Android-Prince-of-Versions",
+        scm        : [
+                connection: "https://github.com/infinum/Android-Prince-of-Versions.git",
+                url       : "https://github.com/infinum/Android-Prince-of-Versions"
+        ]
+]
+ */
+
+task javadocsJar(type: Jar, dependsOn: javadocs) {
+    getArchiveClassifier().set("javadoc")
+    from javadocs.destinationDir
+}
+
+task sourcesJar(type: Jar) {
+    getArchiveClassifier().set("sources")
+    from android.sourceSets.main.java.source
+}
+
+afterEvaluate {
+    publishing {
+        def props = project.ext.mavenPublishProperties
+
+        repositories {
+            maven {
+                name "SonatypeMavenCentral"
+                url props.repository.url
+                credentials {
+                    username props.repository.username
+                    password props.repository.password
+                }
+            }
+        }
+        publications {
+            release(MavenPublication) {
+                from components.release
```

Comment:
This includes dependencies from release scope out of the box, we don't need withXml block anymore